Add load_stac to localprocessing (#445)
Open-EO/openeo-python-client: docs/cookbook/localprocessing.rst

Usage
-----

> Every openEO process graph relies on data, which was always provided by a cloud infrastructure (the openEO back-end) until now.

Reviewer: ```suggestion
Every openEO process graph relies on data which is typically provided by a cloud infrastructure (the openEO back-end).
```

> The client-side processing adds the possibility to read and use local netCDFs, geoTIFFs and ZARR files and remote STAC Collections or Items for your experiments.

Reviewer: ```suggestion
The client-side processing adds the possibility to read and use local netCDFs, geoTIFFs, ZARR files, and remote STAC Collections or Items for your experiments.
```

STAC Collections and Items
~~~~~~~~~~~~~~~~~

.. warning::
    The provided examples using STAC rely on third party STAC Catalogs, we can't guarantee that the urls will remain valid.

With the `load_stac` process it's possible to load and use data provided by remote or local STAC Collections or Items.
The following code snippet loads Sentinel-2 L2A data from a public STAC Catalog, using specific spatial and temporal extent, band name and also properties for cloud coverage.

.. code-block:: pycon

    >>> from openeo.local import LocalConnection
    >>> local_conn = LocalConnection("./")

    >>> url = "https://earth-search.aws.element84.com/v1/collections/sentinel-2-l2a"
    >>> spatial_extent = {"west": 11, "east": 12, "south": 46, "north": 47}
    >>> temporal_extent = ["2019-01-01","2019-06-15"]
    >>> bands = ["red"]
    >>> properties = {"eo:cloud_cover": dict(lt=50)}

>     >>> s2_cube = local_conn.load_stac(url=url,

Reviewer: I'd use a 4 space indent here
